docs/user-guide/commands/argocd_app_delete.md

```diff
@@ -13,8 +13,9 @@ argocd app delete APPNAME [flags]
 ### Options
 
 ```
-      --cascade   Perform a cascaded deletion of all application resources (default true)
-  -h, --help      help for delete
+      --cascade    Perform a cascaded deletion of all application resources (default true)
+  -h, --help       help for delete
+      --noPrompt   Turn off prompting to confirm cascaded deletion of application resources
```

Comment:
CLI args usually follow convention of always being lower case. If you want word separation, usually it is separated by dashes (e.g. `--no-prompt`).

Also, let me think of something better than no-prompt.

Reply:
In many UN*X tools, there is a `--yes|-y` switch - to allow non-interactive use and assume "yes" as answer to prompts.

Reply:
I agree, let's go with `--yes`

Reply:
Hey guys, which do you prefer? ` --yes|y` or `--no-prompt` ?  One advantage of the latter is that it is self-explanatory.

Reply:
Disregard my last comment. I didn't refresh this page.  `--yes` it is.

Reply:
Note that because we're using the Cobra command library, any boolean flag option allows users to specify any of the following combinations:
```
--myflag
--myflag=true
--myflag=false
```

In our case, `myflag` is `yes`, so `--yes=true` and `--yes=false` are actually allowed and I don't currently see any way to disable the last two options such that you will get an invalid argument error.  FWIW, this might suggest that cobra-based boolean flags should have a meaningful name

I've added changes to consider the value of the flag (the right side of the equation) as well, even though the likelihood of a user using these two option formats is low.